### PR TITLE
Task/Update welcome screen UI

### PR DESCRIPTION
**Pull request by kopijunkie**
# Task
The end user should see the new layout for the Welcome screen (see design screenshot below) 

Screenshot 2021-05-26 at 19 32 26

- By default, user should see the default instructions as per Figma design

- Depending on the SDK configuration user should see either document, face or document + face in the instruction text 

- If custom `descriptions` copy for the Welcome screen is supplied by the integrator, the user should see the custom descriptions instead of the default instructions

## Checklist

_put `n/a` if item is not relevant to PR changes_

- [x] Has the CHANGELOG been updated?
- [ ] Has the README been updated?
- [x] Has the CONTRIBUTING doc been updated?
- [ ] Has the RELEASE_GUIDELINES been updated?
- [ ] Has the TESTING_GUIDELINES been updated?
- [ ] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [ ] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [x] Have any new automated tests been implemented or the existing ones changed?
- [ ] Have any new manual tests been written down or the existing ones changed?
- [ ] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

**Review by stefaniacardenas — APPROVED** [review]
Looks good. Do we need to add the strings that changed to the migration file?

File: src/components/Welcome/index.tsx [hunk 1]
```diff
@@ -68,24 +70,41 @@ const WelcomeActions: FunctionComponent<WelcomeActionsProps> = ({
...
+    : ''
 
-  const actions = <WelcomeActions {...{ nextButton, nextStep }} />
-  const welcomeTitle = title ? title : translate('welcome.title')
+  const configuredIDVSteps = steps
+    .filter((step) => step.type === 'document' || step.type === 'face')
+    .map((stepConfig) => stepConfig.type)
+  const welcomeDescriptions = customDescriptions
+    ? customDescriptions
+    : getLocalisedDescriptions(configuredIDVSteps, translate)
```
Comment (stefaniacardenas) [inline src/components/Welcome/index.tsx:102]: What will this display for `poa` or `auth`? Shouldn't we have a default for those cases?

**Review by stefaniacardenas — CHANGES_REQUESTED** [review]

File: src/components/Welcome/index.tsx (hunk 1 above)
Comment (kopijunkie) [reply src/components/Welcome/index.tsx:102]: Good point. I'm not sure about messaging for `auth` or how to preview it so will need to catch up with @carlagn but for `poa` we do need some kind of default messaging in English.

**Comment by kopijunkie** [comment]
> Looks good. Do we need to add the strings that changed to the migration file?

I'm conflicted about whether to add this now or later when all the copy update happens. But could also be worth getting started now.

File: src/components/Welcome/index.tsx (hunk 1 above)
Comment (kopijunkie) [reply src/components/Welcome/index.tsx]: I've mapped the `auth` Welcome screen copy to `face` for now. Auth is currently just using the existing SDK's welcome screen but will bring it up on Slack for discussion with designers.

File: src/locales/de_DE/de_DE.json [hunk 2]
```diff
@@ -476,9 +476,11 @@
...
     "welcome": {
-        "description_p_1": "Um ein Bankkonto zu eröffnen, müssen wir Ihre Identität überprüfen.",
-        "description_p_2": "Es wird nur ein paar Minuten dauern.",
         "next_button": "Identität überprüfen",
-        "title": "Eröffnen Sie Ihr neues Bankkonto"
+        "title": "Identität überprüfen",
+        "subtitle": "It should take a few minutes",
+        "description_p_1": "Use your webcam or phone to photograph:",
+        "description_p_2": "1. your identity document",
+        "description_p_3": "2. your face"
```
Comment (stefaniacardenas) [inline src/locales/de_DE/de_DE.json:484]: Why is this partially translated? Same for the other languages

Comment (kopijunkie) [reply src/locales/de_DE/de_DE.json:484]: I can update them with the latest from Lokalise now, but the description keys all have the localised translations for "Verify Identity". Have raised this question in Slack about the translations.

Comment (kopijunkie) [reply src/locales/de_DE/de_DE.json]: Updated to use the latest key-string for default descriptions after discussion with Rae.

File: src/demo/SdkMount.tsx [hunk 3]
```diff
@@ -17,6 +17,10 @@ import * as Onfido from '../index'
 */
 
 const Onfido = window.Onfido
```
Comment (stefaniacardenas) [inline src/demo/SdkMount.tsx:19]: For Auth you should be able to just do `const Onfido = window.OnfidoAuth` which works fine in the sample app but for some reason it doesn't in this demo app

**Review by stefaniacardenas — APPROVED** [review]

File: src/demo/SdkMount.tsx (hunk 3 above)
Comment (kopijunkie) [reply src/demo/SdkMount.tsx:19]: Interesting. I tried `import * as Onfido from '../index'` after talking to Carla about how to get it running locally & she suggested this, which worked. So just went with that.

**Review by ethan605 — CHANGES_REQUESTED** [review]
Just some minor changes needed

File: src/demo/demoUtils.ts [hunk 4]
```diff
@@ -147,7 +148,22 @@ export const getInitSdkOptions = (): SdkOptions => {
...
+              title: 'Open your new bank account',
+              descriptions: [
+                'To open a bank account, we will need to verify your identity.',
+                'It will only take a couple of minutes.',
+              ],
+              nextButton: 'Verify Identity',
+            },
+          },
+        ]
+      : [{ type: 'welcome' }]
```
[the remark this answers is not in the document]
Comment (kopijunkie) [reply src/demo/demoUtils.ts:166]: Agreed. Looking at the comparison here, the IF-ELSE is easier to read/follow.

**Review by ethan605 — APPROVED** [review]

**Pull request merged**